lib/omise/OmiseSearch.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+require_once dirname(__FILE__).'/res/OmiseApiResource.php';
+
+/**
+ * This class is not intended to be used directly from client code.
+ *
+ * @see OmiseCharge::search()
+ */
+class OmiseSearch extends OmiseApiResource
+{
+    const ENDPOINT = 'search';
+
+    private $dirty = true;
```

Comment:
@iwat I see, then just curious, why you need to check if it's `dirty` in the code, since there has no case that `dirty` can be false?